src/main/java/picard/sam/RevertSam.java

```diff
@@ -257,10 +261,12 @@ protected String[] customCommandLineValidation() {
 
     protected int doWork() {
         IOUtil.assertFileIsReadable(INPUT.toPath());
-        ValidationUtil.assertWritable(OUTPUT, OUTPUT_BY_READGROUP);
+        if (OUTPUT != null && !PicardBucketUtils.isGcsUrl(OUTPUT)) {
```

Comment:
Suggest we add `isFileUrl` to PicardHstPath, and check for that here, since this won't correctly filter paths on the other (non-GCS) cloud providers.